TestCases/species_transport/venturi_primitive_3species/species2_primitiveVenturi_mixingmodel_ND.cfg

```diff
@@ -0,0 +1,145 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: Species mixing with 2 species, i.e. 1 transport equation,  %
+%                   including mixture dependent density, viscosity, thermal    %
+%                   conductivity, heat capacities and mass diffusivity.        %
+%                   The species considered are air and methane                 %
+% Author: Cristopher Morales Ubal                                              %
+% Institution: Eindhoven University of Technology                              %
+% Date: 2022/09/16                                                             %
+% File Version 7.4.0 "Blackbird"                                               %
+%                                                                              %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
+%
+SOLVER= INC_RANS
+KIND_TURB_MODEL= SST
+%
+% ---------------- INCOMPRESSIBLE FLOW CONDITION DEFINITION -------------------%
+%
+INC_DENSITY_MODEL= VARIABLE
+INC_DENSITY_INIT= 1.1766
+%
+INC_VELOCITY_INIT= ( 1.00, 0.0, 0.0 )
+%
+INC_ENERGY_EQUATION= NO
+INC_TEMPERATURE_INIT= 300.0
+%
+INC_NONDIM= INITIAL_VALUES
```

Comment:
I suppose like this the velocity is equal to a dimensional simulation? Have you checked with different reference (initial) values?

Reply:
![ND_checked](https://user-images.githubusercontent.com/98025159/193621744-217beaff-91d6-45e1-ace5-61faff29c5fd.JPG)

Yes I have checked with a different velocity, please let me know if this is ok. Thank you so much in advance :)

Reply:
Yep looks good, if possible make the ND regression exactly equivalent to the dimensional, that way hopefully some of the test values should be very close even before the solver converges.

Reply:
Thank you so much. Yes I will do it :)